Parsers/SymantecProxySG/SymantecProxySG.txt

```diff
@@ -29,7 +29,7 @@
 Syslog
 | where Computer in ("datasource") and Facility == "local0"
```

Comment:
@appelboom, should it not include changes here? 
![ParserChanges](https://user-images.githubusercontent.com/73506651/118560117-d75c0a00-b71d-11eb-8e4c-602dc46a9a52.PNG)

Reply:
Hi, No. They are fields the customer must change. Described in first lines.